docs_outdoornav_user_manual/release_notes.mdx

```diff
@@ -6,6 +6,41 @@ toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
 
+## 0.12.0
+
+### New Features
+-   Modified mission planning modes. Now includes:
+    -   Map mode (which allows the user to create maps of an environment 
+        and send mission using said map)
+    -   Waypoint mode (which allows the user to create more direct mission 
+        by placing waypoints directly on the Aerial overlay)
+-   Added an Assisted teleoperation feature to prevent robot collisions with 
+    obstacles when it is being teleoperated.
+-   Added the upgrade command to the CLI tool. Customers can now upgrade their 
+    software as long as they are eligible to be upgraded to the newest version. 
+    Please contact [Support](./support.mdx) for more information.
+-   New visualization of status command in CLI tool
+-   Added a filebrowser extension to access/view the robot filesystem
+
+### API Features
+
+-   Added new message definitions and service calls for the storage/editing of Maps
+
+### Bug Fixes
+
+-   1973: Fixed python “machineid” error during installation
+-   2089: Fixed the no serial number on /platform/id topic
+-   2169: Fixed the mission manager/scheduler still requesting during Teleop only mode
+
+### Known Issues
+
+-   1396: Robot gets temporarily stuck at start of mission when near a Waypoint
+-   1844: [MPC] Maximum acceleration param doesn't appear to have any effect
+-   2332: upgrading from 0.11 to 0.12.0 in teleoperation only configuration and Q62 
+    installed causes duplication of axis driver. Please contact [Support](./support.mdx)
+    if you run into this issue.
+-   2344: Autonomy not going down during low-power mode
+

```

Comment:
One more item to add, No worries if you miss it but once you're merged in I'll make sure it's there:
2360: Waypoint/Map modes use different mouse click to open context menus